### ISSUE TITLE
Improve user feedback on configuration mistakes

### ISSUE THREAD

**Issue by berndruecker** [zeebe-io/kafka-connect-zeebe#19]
> The only issue I've found with it so far is, the sink can sometimes crash an not recover if you misconfigure the sink.json file.
For example: - i configured it to look for a variable named foo, but forgot to add to the payload from the C# worker.
Crashed the sink. took me a while to debug.

**Comment by npepinpe** [comment]
Yeah I thought about that, but I wasn't sure what was the "best" way to handle errors, i.e. one record fails to be handled, what happens if we keep processing further records? I assume we never process that record. How are users expecting this to behave? Where should we report the error? If you know of Kafka Connect power users who would have insight there, that'd be great!
